src/components/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterType } from '../types/FilterType';
+
+type Props = {
+  todos: Todo[];
+  removeAllTodos: () => void;
+  filter: FilterType;
+  setFilter: React.Dispatch<React.SetStateAction<FilterType>>;
+};
+
+const filterLabels: Record<FilterType, string> = {
+  [FilterType.all]: 'All',
+  [FilterType.active]: 'Active',
+  [FilterType.completed]: 'Completed',
+};
+
+const hrefMap: Record<FilterType, string> = {
+  [FilterType.all]: '#/',
+  [FilterType.active]: '#/active',
+  [FilterType.completed]: '#/completed',
+};
+
+const dataCyMap: Record<FilterType, string> = {
+  [FilterType.all]: 'FilterLinkAll',
+  [FilterType.active]: 'FilterLinkActive',
+  [FilterType.completed]: 'FilterLinkCompleted',
+};
```

Comment:
Don't need to create 3 array, use only `Object.value(FilterType)` for creating one array and you can use slice method inside. If you need some help feel free in the fe_chat